storage-node/src/commands/server.ts

```diff
@@ -157,11 +160,19 @@ Supported values: warn, error, debug, info. Default:debug`,
   async run(): Promise<void> {
     const { flags } = this.parse(Server)
 
-    const logSource = `StorageProvider_${flags.worker}`
+    const api = await this.getApi()
+
+    if (flags.dev) {
+      await this.ensureDevelopmentChain()
+    }
+
+    if (flags.logFilePath && path.relative(flags.logFilePath, flags.uploads) === '') {
+      this.error('Paths for logs and uploads must be unique.')
+    }
```

Comment:
```suggestion
    if (flags.logFilePath && path.relative(flags.logFilePath, flags.uploads) === '..') {
      this.error('Paths for logs and uploads must be unique.')
    }
```

Shouldn't it be `path.relative(flags.logFilePath, flags.uploads) === '..'`, I tested the `path.relative` function as `path.relative('/uploads/temp', '/uploads')` and I got `..` as return value

Reply:
Here is was only trying to protect against the two paths being the same location. I was not concerned with temp or logs being subfolders of the main uploads directory. That is why I was comparing with `''` null string.

Originally I was doing `if (lags.logFilePath === flags.uploads) { }` but I decided to use `path.relative()` to take into account trailing `/` path separator..
 
Given that I added proper filtering of filenames when they are loaded from uploads directory, and if we assume the file names of logs and temp files will not clash with the object id name, perhaps this check is not really necessary at all?